lib/broadway_dashboard.ex

```diff
@@ -104,19 +102,10 @@ defmodule BroadwayDashboard do
     end
   end
 
-  defp nav_pipeline(params) do
+  defp nav_pipeline(params, pipelines) do
     nav = params["nav"]
     nav = if nav && nav != "", do: nav
-
-    if nav do
-      to_existing_atom_or_nil(nav)
-    end
-  end
-
-  defp to_existing_atom_or_nil(nav) do
-    String.to_existing_atom(nav)
-  rescue
-    ArgumentError -> nil
+    nav && Enum.find(pipelines, fn name -> inspect(name) == nav end)
```

Comment:
If it is not an atom, what is the other name? Is it a PID or is it a {:via, ...} tuple?

Reply:
Show on dashboard like this:
```elixir
# atom
iex> Demo.Pipeline |> inspect()
"Demo.Pipeline"

# tuple
iex> {:via, Registry, {MyRegistry, :atom}} |> inspect()
"{:via, Registry, {MyRegistry, :atom}}"

# pid
iex> self() |> inspect()
"#PID<0.3204.0>"
```

Show on url like this:
```elixir
# atom
iex> Demo.Pipeline |> inspect() |> URI.encode_www_form()
"Demo.Pipeline"

# tuple
iex> {:via, Registry, {MyRegistry, :atom}} |> inspect() |> URI.encode_www_form()
"%7B%3Avia%2C+Registry%2C+%7BMyRegistry%2C+%3Aatom%7D%7D"

# pid
iex> self() |> inspect() |> URI.encode_www_form()
"%23PID%3C0.3204.0%3E"
```